### ISSUE TITLE
Add support for `row` query term

### ISSUE THREAD

**Issue by danielcompton** [apa512/clj-rethinkdb#80]
We don't have support for the `row` query term yet. This shouldn't be too difficult to add. @apa512 is there any reason why we don't have it, or has it just not been needed yet?

**Comment by danielcompton** [comment]
`row` doesn't seem to be a standard query term, it's not in QL2.proto. Will need to look at JS driver source to understand what it's doing.

**Comment by apa512** [comment]
Right, just a convenience method that the JS driver uses. I'm not sure that something similar would make sense for Clojure.

**Comment by danielytics** [comment]
In the past, I've simply used functions when in js I would have used row.
It may be convenient to support something similar, but it's easy to work
around not having.
